libs/cart/src/drivers/magento/models/responses/list-shipping-methods.ts

```diff
@@ -0,0 +1,9 @@
+import { MagentoCartShippingMethod } from '../outputs/cart-shipping-method';
+
+export interface ListShippingMethodsResponse {
```

Comment:
I think all Magento model names should be prefixed with "Magento" just to be more clear.

Reply:
Good point, I agree. Especially since these models are being exported.

Reply:
Changed in [`507b798`](https://github.com/graycoreio/daffodil/pull/657/commits/507b798f28097f6219966cc5e8b6c72101b693c4).